src/utils/formatDate.ts

```diff
@@ -1,3 +1,40 @@
 export const formatDate = (date?: string) => {
   return date?.split("T")[0]
 }
+
+export const formatDateRange = (start_date?: string, end_date?: string) => {
```

Comment:
I  also have an update for this. Can you check to see if we can reuse it. [here](https://github.com/nerubia/kh360-react/pull/165/files#diff-3290e87c24e27923abe4a89682818bc0169238f9c4784ab4545dd448c0951415)